packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -91,7 +97,14 @@ const maybeZoomAction = (e?: MouseEvent | null) => e != null && !e.ctrlKey && !e
 /**
  * @alpha
  */
-export const TooltipPlugin2 = ({ config, hoverMode, render, clientZoom = false, queryZoom }: TooltipPlugin2Props) => {
+export const TooltipPlugin2 = ({
+  config,
+  hoverMode,
+  render,
+  clientZoom = false,
+  queryZoom,
+  tooltipOptions,
```

Comment:
this is kind of weird. except for zoom options, the other props are all tooltip options as well.

maybe we can just pass maxWidth and maxHeight as explicit props?

in the future we may rename this plugin to `CursorInteractionPlugin` and then we may want to nest a bit more clearly. like `{tooltip: {maxWidth, maxHeight, hoverMode, render}, zoom: {query, client}}`.

but for now with the current naming, let's keep it flat.